libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/app/SalesforceReactSDKManager.java

```diff
@@ -109,58 +112,68 @@ public static void initReactNative(Context context, Class<? extends Activity> ma
-    	}
+        }
+    }
+
+    @NonNull
+    @Override
+    public String getAppType() {
+        return "ReactNative";
     }
 
-	@Override
-	public String getAppType() {
-		return "ReactNative";
-	}
-
-	/**
-	 * Call this method when setting up ReactInstanceManager
-	 * @return ReactPackage for this application
-	 */
-	public ReactPackage getReactPackage() {
-		return new ReactPackage() {
-			@Override
-			public List<NativeModule> createNativeModules(
-					ReactApplicationContext reactContext) {
-				List<NativeModule> modules = new ArrayList<>();
-				modules.add(new SalesforceOauthReactBridge(reactContext));
-				modules.add(new SalesforceNetReactBridge(reactContext));
-				modules.add(new SmartStoreReactBridge(reactContext));
-				modules.add(new MobileSyncReactBridge(reactContext));
-				return modules;
-			}
-
-			public List<Class<? extends JavaScriptModule>> createJSModules() {
-				return Collections.emptyList();
-			}
-
-			@Override
-			public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
-				return Collections.emptyList();
-			}
-		};
-	}
-
-	@Override
-	protected LinkedHashMap<String, DevActionHandler> getDevActions(final Activity frontActivity) {
-		LinkedHashMap<String, DevActionHandler> devActions = super.getDevActions(frontActivity);
-		devActions.put(
-				"React Native Dev Support", new DevActionHandler() {
-					@Override
-					public void onSelected() {
-						((SalesforceReactActivity) frontActivity).showReactDevOptionsDialog();
-					}
-				});
-
-		return devActions;
-	}
+    /**
+     * Call this method when setting up ReactInstanceManager
+     *
+     * @return ReactPackage for this application
+     */
+    public ReactPackage getReactPackage() {
+        return new ReactPackage() {
+
+            @NonNull
+            @Override
+            public List<NativeModule> createNativeModules(
+                    @NonNull ReactApplicationContext reactContext
+            ) {
+                List<NativeModule> modules = new ArrayList<>();
+                modules.add(new SalesforceOauthReactBridge(reactContext));
+                modules.add(new SalesforceNetReactBridge(reactContext));
+                modules.add(new SmartStoreReactBridge(reactContext));
+                modules.add(new MobileSyncReactBridge(reactContext));
+                return modules;
+            }
+
+            /** @noinspection unused*/
+            public List<Class<? extends JavaScriptModule>> createJSModules() {
+                return Collections.emptyList();
+            }
+
+            @NonNull
+            @Override
+            public List<ViewManager> createViewManagers(
+                    @NonNull ReactApplicationContext reactContext
+            ) {
+                return Collections.emptyList();
+            }
+        };
+    }
+
+    @NonNull
+    @Override
+    protected Map<String, DevActionHandler> getDevActions(
```

Comment:
Everything in this source is inspector cleanup for existing trivial issues and those introduced by the Kotlin conversion.  The one exception is this change from a Collections implementation to an interface type from the Kotlin conversion.  This is probably a best practice update anyways.